data/redirects.yml

```diff
@@ -1,338 +1,559 @@
+  url: "/changelog/logentries-and-sumo-logic-setup-now-a-breeze/"
+- loc: "/blog/logs-v2/"
+  url: "/changelog/logs-v2/"
+- loc: "/blog/logs_v2.html"
+  url: "/changelog/logs-v2/"
+- loc: "/blog/managed-https-endpoints-now-support-internal-endpoints/"
+  url: "/changelog/managed-https-endpoints-now-support-internal-endpoints/"
+- loc: "/blog/managed-https/"
+  url: "/changelog/managed-https/"
+- loc: "/blog/managed_https.html"
+  url: "/changelog/managed-https/"
+- loc: "/blog/mfa.html"
+  url: "/changelog/mfa/"
+- loc: "/blog/mfa/"
+  url: "/changelog/mfa/"
+- loc: "/blog/most_popular_databases.html"
+  url: "/blog/most-popular-databases/"
+- loc: "/blog/most_popular_languages_in_digital_health.html"
+  url: "/blog/most-popular-languages-in-digital-health/"
+- loc: "/blog/no-flash-no-problem.html"
+  url: "/blog/no-flash-no-problem/"
+- loc: "/blog/no_flash_no_problem.html"
+  url: "/blog/no-flash-no-problem/"
+- loc: "/blog/on-demand-backups/"
+  url: "/changelog/on-demand-backups/"
+- loc: "/blog/on_demand_backups.html"
+  url: "/changelog/on-demand-backups/"
+- loc: "/blog/pride_app.html"
+  url: "/blog/pride-app/"
+- loc: "/blog/rabbitmq-management/"
+  url: "/changelog/rabbitmq-management/"
+- loc: "/blog/redis-ssl/"
+  url: "/changelog/redis-ssl/"
+- loc: "/blog/reinvent_2014.html"
+  url: "/blog/reinvent-2014/"
+- loc: "/blog/smoother_ember_upgrades.html"
+  url: "/blog/smoother-ember-upgrades/"
+- loc: "/blog/what-is-a-hipaa-baa/"
+  url: "/resources/what-is-a-hipaa-baa/"
+- loc: "/blog/what_is_a_hipaa_baa.html"
+  url: "/resources/what-is-a-hipaa-baa/"
+- loc: "/blog/windows-cli/"
+  url: "/changelog/windows-cli/"
+- loc: "/blog/y_combinator.html"
+  url: "/blog/y-combinator/"
+- loc: "/company/careers/"
+  url: "/company/"
+- loc: "/company/press/"
+  url: "/media/"
+- loc: "/company/resources/index.html"
+  url: "/media/"
+- loc: "/compliance/"
+  url: "/gridiron/"
+- loc: "/docs/"
+  url: "/documentation/index.html"
+- loc: "/legal/acceptable_use.html"
+  url: "/legal/acceptable-use/"
+- loc: "/legal/breach_indemnification_agreement.html"
+  url: "/legal/breach-indemnification-agreement/"
+- loc: "/legal/fees_and_pricing.html"
+  url: "/legal/fees-and-pricing/"
+- loc: "/legal/privacy.html"
+  url: "/legal/privacy/"
+- loc: "/legal/responsible_disclosure.html"
+  url: "/legal/responsible-disclosure/"
+- loc: "/legal/security.html"
+  url: "/legal/security/"
+- loc: "/legal/service_level_agreement.html"
+  url: "/legal/service-level-agreement/"
+- loc: "/legal/support.html"
+  url: "/legal/support/"
+- loc: "/legal/terms_of_service.html"
+  url: "/legal/terms-of-service/"
+- loc: "/legal/trademark.html"
+  url: "/legal/trademark/"
+- loc: "/resources/common-hipaa-questions/index.html"
+  url: "/learn/hipaa-compliance-for-developers-getting-started/"
+- loc: "/resources/common_HIPAA_questions/index.html"
+  url: "/learn/hipaa-compliance-for-developers-getting-started/"
+- loc: "/support/"
+  url: "/documentation/index.html"
```

Comment:
The links Sphinx generates internally use `index.html`, so this would be inconsistent with 2 ways to access the same page.

More importantly, Sphinx expects you to access the page at `index.html`, so I'd rather not do something different behind Sphinx's back (so to speak): historically, doing so has led us to a number of broken links when moving between dev and prod (https://github.com/aptible/www.aptible.com/pull/482/files). This is especially important considering Sphinx generates relative links for everything (which _is_ good for us because it lets us easily anchor the docs at `/documentation`, though).

As an aside: there is only one `index.html`: the documentation home page (which is, literlaly, "the index"), so I dont thyink this is very a big deal.